layouts/partials/minifi-topbar.html

```diff
@@ -16,30 +16,32 @@ <h1>
             <!-- Right Nav Section -->
             <ul class="right">
                 <li class="has-dropdown">
-                    <a href="#">Documentation</a>
+                    <a href="#">MiNiFi Java</a>
                     <ul class="dropdown">
-                        <li><a href="getting-started.html"><i class="fa"></i>Getting Started</a></li>
-                        <li><a href="minifi-java-agent-quick-start.html"><i class="fa"></i>MiNiFi Java Quick Start Guide</a></li>
-                        <li><a href="system-admin-guide.html"><i class="fa"></i>Administrator's Guide</a></li>
-                        <li><a href="minifi-toolkit.html"><i class="fa"></i>Converter Toolkit</a></li>
-                        <li><a href="faq.html"><i class="fa"></i>FAQ</a></li>
-                        <li><a href="https://cwiki.apache.org/confluence/display/MiNiFi"><i class="fa fa-external-link external-link"></i>Wiki</a></li>
+                        <li><a href="minifi-java-agent-quick-start.html"><i class="fa"></i>Quick Start</a></li>
+                        <li><a href="minifi-java-toolkit.html"><i class="fa"></i>Converter Toolkit</a></li>
+                        <li><a href="minifi-java-system-admin-guide.html"><i class="fa"></i>Administrator's Guide</a></li>
+                        <li><a href="minifi-java-download.html"><i class="fa fa-download external-link"></i>Download</a></li>
                     </ul>
                 </li>
                 <li class="has-dropdown">
-                    <a href="#">Downloads</a>
+                    <a href="#">MiNiFi C++</a>
                     <ul class="dropdown">
-                        <li><a href="download.html">Download MiNiFi Components</a></li>
-                        <li><a href="https://cwiki.apache.org/confluence/display/MINIFI/Release+Notes"><i class="fa fa-external-link external-link"></i>MiNiFi Java Release Notes</a></li>
-                        <li><a href="https://cwiki.apache.org/confluence/display/MINIFI/Release+Notes#ReleaseNotes-MiNiFi(C++)"><i class="fa fa-external-link external-link"></i>MiNiFi C++ Release Notes</a></li>
+                        <li><a href="https://github.com/apache/nifi-minifi-cpp?tab=readme-ov-file#installing-as-a-service"><i class="fa fa-external-link external-link"></i>Installation</a></li>
+                        <li><a href="https://github.com/apache/nifi-minifi-cpp/blob/main/CONFIGURE.md"><i class="fa fa-external-link external-link"></i>Configuration</a></li>
+                        <li><a href="https://github.com/apache/nifi-minifi-cpp/blob/main/PROCESSORS.md"><i class="fa fa-external-link external-link"></i>Supported Processors</a></li>
```

Comment:
Recommend shortening this to just `Processors` as the label.
```suggestion
                        <li><a href="https://github.com/apache/nifi-minifi-cpp/blob/main/PROCESSORS.md"><i class="fa fa-external-link external-link"></i>Processors</a></li>
```